modular_exponential/modular_exponential.py

```diff
@@ -0,0 +1,21 @@
+def mod_exponent(base_, pow_, mod_):
```

Comment:
Since you have now modified this code according to skeleton code, you will not be facing issues of variable shadowing.